internal/handlers/create_catalog.go

```diff
@@ -107,7 +107,9 @@ func (h *CreateCatalogHandler) Handle(message messaging.Message) error {
 
 		images, err := h.refToImages(registryClient, ref, registry)
 		if err != nil {
-			return fmt.Errorf("cannot get images for %s: %w", ref, err)
+			h.logger.Error("Error cannot get images for ref", "ref", ref, "error", err)
```

Comment:
nit, "Error" would already be part of the produced output

```suggestion
			h.logger.Error("cannot get images for ref", "ref", ref, "error", err)
```

Reply:
Hi @flavio , 
After dig into the unit-test, I still not sure where to put the cases fir this PR
Any good suggestions?

Reply:
inside of `create_catalog_test.go`. Take inspiration from something like `TestCreateCatalogHandler_Handle`.

I would create a new test, mock a repository with a container image and an OCI artifact inside of it

Reply:
Hi @flavio ,
I noticed that our current code may have issues when processing multiple platforms in a single image. For example, if we encounter a list like [arm64, amd, unknown (which causes a failure), other-platform], our code stops processing after the failure ("unknown"), and we miss generating images for subsequent platforms.

Should we adjust the logic to handle such errors gracefully, so that a failure on one platform doesn’t block processing of the others?

Reply:
> Should we adjust the logic to handle such errors gracefully, so that a failure on one platform doesn’t block processing of the others?

For sure. I leave up to you to decided whether we should fix that with another PR. If you want to fix that with another PR, please write a new issue with a detailed explanation. When fixing the issue also provide the unit tests for that

Reply:
I will also fix this within the same pr with next commit